ICVL.py

```diff
@@ -23,7 +23,7 @@
 # Local files
 from model_objects import model_parameters
 from DataSetInfoAbstractClass import dataSetInfoAbstract
-import cPickle
+import pickle
```

Comment:
can remove this - doesn't look like it needs to be imported twice! (see line 17)